andela/travel_tool_front
# #161660191 bug fix input fields

#### What does this PR do?
Fixes issue with input fields not being editable


#### Description of Task to be completed?
Update the yarn lock file

#### How should this be manually tested?
- Clone the repo `https://github.com/andela/travel_tool_front.git`
- Checkout to the branch:
`bug-fix-input-fields-161660191`
- Install dependencies:
`yarn install`
- Start the application:
```yarn start```

- Login to the application
- Navigate to the request page
- Click the make a new request button
- You should be able to edit the input fields

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[161660191](https://www.pivotaltracker.com/story/show/161660191)

#### Screenshots (if appropriate)
N/A

#### Questions:
